examples/error-demo/README.md

```diff
@@ -0,0 +1,7 @@
+# Run locally
+`npm start`
+
+# Open in browser
+[Click here](https://codesandbox.io/s/github/redux-saga/redux-saga/tree/master/examples/error-demo)
+
+*Note* babel-plugin doesn't work yet on codesanbox.
```

Comment:
do you know what's the problem? i guess codesandbox doesnt support babel customization?